include/swift/AST/PropertyWrappers.h

```diff
@@ -170,16 +170,36 @@ void simple_display(llvm::raw_ostream &os, PropertyWrapperLValueness l);
 /// be initialized out-of-line using an expression of the wrapped property type.
 PropertyWrapperValuePlaceholderExpr *findWrappedValuePlaceholder(Expr *init);
 
-/// Describes the backing property of a property that has an attached wrapper.
-struct PropertyWrapperBackingPropertyInfo {
+/// The synthesized auxiliary declarations for a wrapped property.
```

Comment:
It might be worth specifying what are the auxiliary variables here to make it easier to understand without looking at the code.

Reply:
Done!

Reply:
Looks great!